Code/GraphMol/MolDraw2D/DrawMol.cpp

```diff
@@ -3370,6 +3370,19 @@ void DrawMol::makeHighlightEnd(const Atom *end1, const Atom *end2,
     // There is only 1 intersection to deal with, which is easier - just
     // a slanted end.
     auto end3Cds = atCds_[end1HighNbrs[0]->getIdx()];
+    auto b1 = end2Cds.directionVector(end1Cds);
+    auto b2 = end2Cds.directionVector(end3Cds);
+    if (1.0 - fabs(b1.dotProduct(b2)) < 1.0e-4) {
+      // rotate end3 about end2 by a small amount to create
+      // an inner and outer
+      end3Cds -= end2Cds;
+      RDGeom::Transform2D trans;
+      trans.SetTransform(Point2D(0, 0), M_PI / 20.0);
+      trans.TransformPoint(end3Cds);
+      end3Cds += end2Cds;
```

Comment:
I haven't tested extensively, but wouldn't this be simpler/more efficient if you just offset end3Cds a short distance along the perpendicular?
```suggestion
      auto d32 = end3Cds - end2Cds;
      Point2D d32transp(d32.y, -d32.x);
      d32transp *= 0.1;
      end3Cds += d32transp;
```

Reply:
Originally I thought I'd have to move the new ends back the other way, hence the rotation.  That turned out not to be necessary.
I don't think efficiency is a big concern because this isn't going to happen very often.  Having to do the dot-product every time just in case this unusual thing might have cropped up has had more of an impact on efficiency, I would imagine.  Still, there's no harm in keeping it simpler.